Views/CarouselItem-ResponsiveMediaField.liquid

```diff
@@ -0,0 +1,11 @@
+﻿{% if Model.HasMedia %}
+    {% for item in Model.Media %}
+        <picture class="card__background">
+            {% for source in item.Sources %}
+                <source srcset="{{ source.Url }}" media="(min-width: {{ source.Breakpoint }}px)" />
+            {% endfor %}
+
+            <img src="{{ item.SmallestImageUrl }}" />

```

Comment:
Can we add `alt="{{ item.MediaText }}` here?

Reply:
Might also be worth making `loading="lazy"` the default here?

Reply:
And it would be worth adding a ResponsiveImage-ResponsiveMediaField.liquid so we can set these for Widget-ResponsiveImage too

Reply:
Don't think `ResponsiveImage-ResponsiveMediaField.liquid` is needed as it doesn't need to deviate from the default markup? I've added a change within the fields module to use `loading="lazy"` - https://github.com/EtchUK/Etch.OrchardCore.Fields/pull/72